velox/common/memory/MemoryArbitrator.cpp

```diff
@@ -60,34 +60,100 @@ class Registry {
   std::unordered_map<std::string, MemoryArbitrator::Factory> map_;
 };
 
-Registry registry;
+Registry& registry() {
```

Comment:
Shall we change the name to arbitratorFactories() to align with factories in the codebase?